src/components/themed/Tile.js

```diff
@@ -58,12 +58,14 @@ class TileComponent extends React.PureComponent<Props> {
 
 const getStyles = cacheStyles((theme: Theme) => ({
   container: {
-    width: '100%',
-    backgroundColor: theme.tileBackground,
```

Comment:
This should still use the tile background theme. Just change the theme color to transparent

Reply:
fixed